src/model/constraints/placement_constraints.py

```diff
@@ -23,6 +23,12 @@ def get_heuristic_value(self):
         return maxDist
 
 def get_region_from_str(region_str):
-    if region_str == 'Junction':
+    if region_str == 'Drivable':
+        return Drivable_Type()
+    elif region_str == 'Road':
+        return Road_Type()
+    elif region_str == 'Junction':
         return Junction_Type()
-    # TODO: Map other region(type)s
+    else:
+        pass
+        # TODO: Handle instances with IDs
```

Comment:
What should we do if we are given a region ID (instance, instead of type)? I guess we should query the map somehow and throw an error if the ID doesn't exist?

Reply:
Yes. But I am thinkin this will happen later in the process. I do do something similar for the `maneuverJunction` parameter. I should be able to implement this for the placement constraint as well.